core/src/main/java/org/apache/iceberg/hadoop/HadoopCatalog.java

```diff
@@ -104,6 +107,8 @@ public void initialize(String name, Map<String, String> properties) {
     String fileIOImpl = properties.get(CatalogProperties.FILE_IO_IMPL);
     this.fileIO = fileIOImpl == null ? new HadoopFileIO(conf) : CatalogUtil.loadFileIO(fileIOImpl, properties, conf);
 
+    this.lockManager = LockManagers.from(properties);
```

Comment:
@CodingCat, the lock manager API has a `close` method that I think should be called when the catalog is closed. Can you implement closing similar to what's in the `GlueCatalog`?

Reply:
done